test-support/helpers/ember-file-upload.js

```diff
@@ -0,0 +1,5 @@
+import upload from './upload';
+
+export {
```

Comment:
This export might be unnecessary, since we're importing `./upload` for side-effects only.

Reply:
Sounds right- I think that I didn't do this because Ember's testing story is moving away from globals. But since I have no idea where that's in the pipeline of things, it makes sense to use what's provided out-of-the box.